backend/core/serializers.py

```diff
@@ -174,6 +186,7 @@ class Meta:
 
 class ThreatReadSerializer(BaseModelSerializer):
     folder = FieldsRelatedField()
+    library = LibraryReadSerializer()
```

Comment:
Serializing the entirety of each threat's library is overkill, as only their names and urns are needed.